src/local-development/create-remote-component.ts

```diff
@@ -1,49 +1,142 @@
 import { AxiosRequestConfig } from 'axios';
-import { AppComponentMetadata, LocalAppOriginWithSecret } from './types/makecomapp.types';
```

Comment:
When importing just types, I'd prefer sticking to the import type syntax to make it more apparent.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

Reply:
Done